### DISCUSSION TITLE
"NOT" Logic on Filter

### DISCUSSION THREAD

**Discussion by cainezen** [civitai/civitai#336] [Ideas]
Can we please get some kind of "NOT" logic to assign to a filter?  Given the amount of Anime-centric content on the platform, being able to say ("NOT" Anime) would be greatly appreciated.

**Comment by jndietz** [comment]
You can exclude tags from the entire site via your profile:

![image](https://user-images.githubusercontent.com/4515791/224220743-ece9f15a-c850-4d6b-8341-e8172cb25a65.png)

But I still think this request has value.  With the large influx of models, it would be good to curate your results in a friendlier fashion.  I'll start on that.  I was thinking that a search could be something like `women real person -anime` to exclude anime.  Anything prefixed with the `-` would then be passed to the `not` in the where clause.  I'll have to see how the tags are related to the models, but that's my hypothesis for now...

**Comment by michaelbrave** [comment]
Second this, would be nice to search LORA's but removing character/celebrity ones, a toggle of multiple things to remove would be ideal. Meaning I don't want to turn it off the profile, just for that specific search.

**Comment by blackshot** [comment]
May #351 be of interest, make some noise for being merge 😁

**Reply by WASasquatch** [reply]
No reason for categories when you have tagging imo. It's literally what tags is about, but being dynamic and allowing overlap. It just needs improvement, such as search based exclusion (not from your profile). The issues with categories I can immediately see is a good set of categories. The examples you provide for your merge are kinda random -- and more like tags.
